test/verify/check-storage-nfs

```diff
@@ -32,6 +33,8 @@ class TestStorageNfs(StorageCase):
         self.login_and_go("/storage")
 
         m.execute("mkdir /home/foo /home/bar")
+        self.addCleanup(m.execute, "systemctl restart nfs-server")
+        self.restore_file("/etc/exports")
```

Comment:
restore_file() has a `post_restore_action` for that purpose; reads a little nicer to keep things together which belong together.